price/src/price_service.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{price_generator::MessageReceiver, GrpcResult};
```

Comment:
Delete this file since we're not exposing GRPC services

Reply:
Removed in [a454294](https://github.com/helium/oracles/pull/400/commits/a454294be181d1c7e47fa93265e510712be6cdc0)